src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -141,6 +142,10 @@
             <argument type="service" id="request_stack" />
         </service>
 
+        <service id="sylius.theme.context.channel_based" class="%sylius.theme.context.channel_based.class%" public="false">
```

Comment:
There's only one service related to theme bundle registered, but that's a good idea to extract checkout steps, order processors, settings schemas, etc.